modules/contentbox/modules/contentbox-ui/interceptors/CBRequest.cfc

```diff
@@ -107,17 +107,24 @@ component extends="coldbox.system.Interceptor" {
 			}
 		}
 
+		var linkLogout = "#prc.cbAdminEntryPoint#/security/doLogout";
+
 		// Render the admin bar out
 		var adminBar = renderView(
 			view   = "adminbar/index",
 			module = "contentbox-ui",
 			args   = {
 				oContent       : oContent ?: javacast( "null", "" ),
 				linkEdit       : linkEdit,
+				linkLogout     : linkLogout,
 				oCurrentAuthor : prc.oCurrentAuthor
 			}
 		);
-		cfhtmlhead( text = "#adminBar#" );
+		var adminBarCSS = renderView( view = "adminbar/adminbarCSS", module = "contentbox-ui" );
+		buffer.append( "#adminBar#" );
```

Comment:
Same here, avoid creation of variables that are only used to be passed to a function.  Add them inline.

Reply:
Done.